mmcv/utils/registry.py

```diff
@@ -139,9 +139,10 @@ def infer_scope():
         Returns:
             str: The inferred scope name.
         """
-        # inspect.stack() trace where this function is called, the index-2
-        # indicates the frame where `infer_scope()` is called
-        filename = inspect.getmodule(inspect.stack()[2][0]).__name__
```

Comment:
Suggesting leave a pr tag here as comment, such as:
```
pr #1844 change ``filename = inspect.getmodule(inspect.stack()[2][0]).__name__`` to below to speed up this code segment. 
```

Reply:
Sure, I've added a comment with a reference to the PR